app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -81,6 +83,16 @@ export class AddForm extends React.Component {
     this.setState({ formType: formType.trim() });
   }
 
+  checkIfDrugHasActiveOrder = (drug) => {
+    const { filteredOrders } = this.props.fetchOrdersReducer;
+    const drugOrders = filteredOrders.filter(order => order.type === 'drugorder');
+    const findDrug = drugOrders.filter(order => order.drug.display === drug);
+    if (findDrug.length > 0) {
```

Comment:
I think since a user cant order for the same drug more than once,  exactly equal to 1 would be more discrete. `>0` takes on a situation where the same order is active 100 times.

Reply:
This makes sense. Will update it

Reply:
Done